Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 Django = "==3.2.6"
 psycopg2-binary = "==2.9.1"
+django-model-utils = "*"
```

Comment:
[_suggestions, blocking_]: I notice a new library introduced in the last round of changes. Two notes/suggestions here:

1. Importing the library with an asterisk in place a version means that no version is being pinned. This means that updating our dependencies can introduce breaking changes without us knowing. Let's pin to the latest major version when we introduce a new library. The Eng Practices Guide has a little bit about pinning dependencies here: https://engineering.18f.gov/python/.
2. It looks like the only import from this library is `Choices`. The [django-model-utils docs for Choices](https://django-model-utils.readthedocs.io/en/latest/utilities.html#choices) say: "Django 3.0 adds enumeration types. These provide most of the same features as Choices." Our project is using Django 3.2.6, so I would suggest we use Django-provided built-in `Choices` rather than a module from an external library. If there are specific features we need from django-model-utils that we can't get straight from Django, let's document those. Otherwise, the fewer external libraries we use the better off we are in terms of both security and complexity. 

Summary: I would suggest we remove the library if possible and use built-in Django features instead. If the library is needed, let's pin to a major version.

Reply:
Ok sounds good. I didn't know about the enumeration types in Django 3.0. Will make those changes now.

Reply:
@alexsoble so I just added the list of choices tuples back in that was there in the original code for the PR and moved them inside the relevant class. I was using `Choices` from the imported library to try and simplify the code (AKA not need a tuple for a single choice, and instead just a single string) but I understand your concern about unnecessary libraries. If I were to use an Enumeration class such as TextChoices, IMO the benefits do not outweigh the overly complicatedness and, also, the TextChoices class gives me less control over how things are displayed in the UI, i.e. Django automatically uppercases the first letter and lower cases the rest so what should display as `OPRE Appropriation` displays as `Opre Appropriation` in the UI and it's more complicated to overwrite this than it's worth. So back to the simple way!

Reply:
@carjug Gotcha! I haven't done a deep dive and might be wrong, but it looks like [TextChoices](https://docs.djangoproject.com/en/3.0/ref/models/fields/#enumeration-types) could give you more control over how choices are displayed, by surfacing a human readable `label` option? 

I'll marked this as resolved for now, since I am happy with any solution that uses either no new libraries, or a new library pinned to a specific version. 

Thank you for implementing this feedback @carjug!

Reply:
Yeah I tried using the `label` option, but it was not working as one would expect.